docker/hq-compose-services.yml

```diff
@@ -56,7 +45,7 @@ services:
       ES_CLUSTER_NAME: ES_CLUSTER_NAME
       ES_JAVA_OPTS: "-Des.script.engine.groovy.inline.aggs=true -Des.script.engine.groovy.inline.search=true"
     ports:
-      - "6200:6200" # is used in python-elasticsearch-v7 build matrix
+      - "9200:9200" # is used in python-elasticsearch-v7 build matrix
```

Comment:
looks like this comment is not valid any more?

Reply:
oops, good catch, thanks

Reply:
https://github.com/dimagi/commcare-hq/pull/29647